LPS-125185 Remove mentions of metal in .npmbundlerrc file within document-library

kresimir-coko opened the pull request:
Fixes [LPS-125185](https://issues.liferay.com/browse/LPS-125185).

I played around with doing the same with the remaining 2 modules that have `metal` mentions (`frontend-taglib-clay` & `frontend-taglib-chart`). I deduced that removing mentions from `frontend-taglib-clay` didn't cause any errors in my quick tests (clicking around a few modules), but removing mentions from `frontend-taglib-chart` and dropping a `Chart` portlet into the landing page triggers the following error:
```bash
2021-03-22 11:31:20.238 INFO  [fileinstall-directory-watcher][BundleStartStopLogger:49] STOPPED com.liferay.frontend.taglib.chart_4.0.0 [271]
2021-03-22 11:31:20.364 INFO  [Refresh Thread: Equinox Container: da509087-e1cd-4c59-adc9-ef15535fc560][BundleStartStopLogger:49] STOPPED com.liferay.frontend.taglib.chart.sample.web_1.0.0 [1035]
2021-03-22 11:31:20.449 ERROR [fileinstall-directory-watcher][DirectoryWatcher:1156] Unable to start bundle: file:/Users/kresimircoko/Documents/Work/Projects/dxp/bundles/osgi/modules/com.liferay.frontend.taglib.chart.sample.web.jar
org.osgi.framework.BundleException: Could not resolve module: com.liferay.frontend.taglib.chart.sample.web [1035]_  Unresolved requirement: Import-Package: com.liferay.frontend.taglib.chart.model; version="[1.0.0,2.0.0)"_ [Sanitized]
        at org.eclipse.osgi.container.Module.start(Module.java:444)
        at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
        at com.liferay.portal.file.install.internal.DirectoryWatcher._startBundle(DirectoryWatcher.java:1139)
        at com.liferay.portal.file.install.internal.DirectoryWatcher._startBundles(DirectoryWatcher.java:1172)
        at com.liferay.portal.file.install.internal.DirectoryWatcher._startAllBundles(DirectoryWatcher.java:1117)
        at com.liferay.portal.file.install.internal.DirectoryWatcher._process(DirectoryWatcher.java:1029)
        at com.liferay.portal.file.install.internal.DirectoryWatcher.run(DirectoryWatcher.java:272)
2021-03-22 11:31:41.283 INFO  [Refresh Thread: Equinox Container: da509087-e1cd-4c59-adc9-ef15535fc560][BundleStartStopLogger:46] STARTED com.liferay.frontend.taglib.chart.sample.web_1.0.0 [1035]
2021-03-22 11:31:41.427 INFO  [fileinstall-directory-watcher][BundleStartStopLogger:46] STARTED com.liferay.frontend.taglib.chart_4.0.0 [1156]
```

So I decided to just push a commit that removes `metal` mentions from `document-library-web/.npmbundlerrc`.
If you think I should add a commit that removes `metal` from `frontend-taglib-clay` to test out yourself, let me know!

/cc @liferay-lima

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-8.liferay.com/job/test-portal-source-format/4474/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-8.liferay.com//userContent/jobs/test-portal-source-format/builds/4474/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest(master)/7833/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-5.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7833/jenkins-report.html">here</a>.</b></p>

wincent commented:
> So I decided to just push a commit that removes `metal` mentions from `document-library-web/.npmbundlerrc`.
> If you think I should add a commit that removes `metal` from `frontend-taglib-clay` to test out yourself, let me know!

I think just doing it in `document-library-web` is the right call here. @jbalsas said this on the LPS about 6 days ago:

> Based on the current scope adjustment we're doing, this should be limited most likely to remove the `.npmbundlerrc` file inside the `document-library-web` module which is the only one that truly has no metal dependencies.

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 6 out of 9 jobs passed</h3><h3>:x: ci:test:relevant - 13 out of 23 jobs passed in 2 hours 14 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/00d824101d48ef815d0190a64a9722205b721666">00d824101d48ef815d0190a64a9722205b721666</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/00d824101d48ef815d0190a64a9722205b721666">00d824101d48ef815d0190a64a9722205b721666</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//1663">Acceptance Upstream DXP (master) #1663</a></p><details><summary><strong>ci:test:stable - 6 out of 9 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/467566/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352899/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</strong></strike></a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/380797/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</strong></strike></a></li></ul></div><details><summary><strong>6 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352898/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/380798/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/467565/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352906/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352901/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457620/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 13 out of 23 jobs PASSED</strong></summary><div><h4>10 Failed Jobs:</h4><ul><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest(master)/7833/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457621/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/380800/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/332066/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352902/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352900/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</strong></strike></a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/467566/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352907/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352899/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</strong></strike></a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/380797/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</strong></strike></a></li></ul></div><details><summary><strong>13 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352905/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352903/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352898/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/380798/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/467565/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/380801/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352904/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/380796/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352906/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/380799/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352901/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457620/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352897/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-5.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7833/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457621/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>6 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/457621//consoleText">AXIS_VARIABLE=0,label_exp=!master #457621</a><pre><code>Finished: FAILURE</code></pre></div></li></ol></div></li><li><div><h5><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352899/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>0 Tests Failed.</p></div><div><p>Irrecoverable Jenkins Error: <a href="https://issues.liferay.com/browse/LRCI-1422">ClosedChannelException</a></p><pre><code>FATAL: java.nio.channels.ClosedChannelException
java.nio.channels.ClosedChannelException
Also:   hudson.remoting.Channel$CallSiteStackTrace: Remote call to JNLP4-connect connection from cloud-host-772.lax.liferay.com/172.16.187.62:38058
		at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1743)
		at hudson.remoting.Request.call(Request.java:202)
		at hudson.remoting.Channel.call(Channel.java:956)
		at hudson.Launcher$RemoteLauncher.kill(Launcher.java:1085)
		at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:510)
		at hudson.model.Run.execute(Run.java:1816)
		at hudson.matrix.MatrixBuild.run(MatrixBuild.java:323)
		at hudson.model.ResourceController.execute(ResourceController.java:97)
		at hudson.model.Executor.run(Executor.java:429)</code></pre></div></div></li><li><div><h5><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/380797/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/380797//consoleText">AXIS_VARIABLE=0,label_exp=!master #380797</a><pre><code>stop-current-job:
     [echo] stop.current.job.start.timestamp: 03-23-2021 10:54:12:242 PDT
[beanshell] Retrying https://testray.liferay.com/reports/production/logs/test-1-5/1616415869504/test-portal-acceptance-pullrequest%28master%29/7833/unit-jdk8_stable/0/0/ in 5 seconds
[beanshell] Retrying https://testray.liferay.com/reports/production/logs/test-1-5/1616415869504/test-portal-acceptance-pullrequest%28master%29/7833/unit-jdk8_stable/0/0/ in 5 seconds
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/json.data
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-jenkins-ee
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210323105422967
     [exec] Result: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210323105423006
     [exec] Killing Chrome.
     [exec] Killing Firefox.
     [exec] chrome: no process found
     [exec] firefox: no process found
     [exec] chrome: no process found
     [exec] firefox: no process found
     [exec] Result: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210323105423043
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210323105423069
     [exec] HEAD is now at 00d824101d48 LPS-129442 sf
     [exec] Already on 'master'
     [exec] Already on 'master'
     [exec] HEAD is now at 425c4c28 minor improvements
     [exec] Already on '2.x'
     [exec] Already on '2.x'
     [exec] HEAD is now at 434358cb4 FACES-3558 Upgrade to com.liferay.faces.test.selenium-0.5.5.jar
     [exec] Already on '4.x'
     [exec] Already on '4.x'
     [exec] HEAD is now at f7819860 [maven-release-plugin] prepare release 4.0.0
     [exec] Already on '4.x'
     [exec] Already on '4.x'
[stopwatch] [stop.current.job: 13.762 sec]

BUILD SUCCESSFUL
Total time: 14 seconds
+ echo 'Exit shell.'
Exit shell.
[EnvInject] - Injecting environment variables from a build step.
[EnvInject] - Injecting as environment variables the properties file path '/opt/dev/projects/github/liferay-jenkins-ee/stop.properties'
[EnvInject] - Variables injected successfully.
[description-setter] Description set: &lt;strong&gt;liferay-frontend&lt;/strong&gt; - &lt;a href="https://github.com/liferay-frontend/liferay-portal/pull/923"&gt;PR#923&lt;/a&gt; - &lt;a href="https://test-1-5.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7833/jenkins-report.html"&gt;Jenkins Report&lt;/a&gt;
Finished: FAILURE</code></pre></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 00d824101d48ef815d0190a64a9722205b721666:</strong></summary><ol><li><div><h5><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/380800/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/380800//consoleText">AXIS_VARIABLE=0,label_exp=!master #380800</a><pre><code>     [exec] Result: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210323072242733
     [exec] Killing Chrome.
     [exec] Killing Firefox.
     [exec] chrome: no process found
     [exec] firefox: no process found
     [exec] chrome: no process found
     [exec] firefox: no process found
     [exec] Result: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210323072242766
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210323072242904
     [exec] HEAD is now at 00d824101d48 LPS-129442 sf
     [exec] Already on 'master'
     [exec] Already on 'master'
     [exec] HEAD is now at 6fe760c2 Merge pr-119
     [exec] Switched to branch '2.x'
     [exec] Switched to branch '2.x'
     [exec] HEAD is now at 5570977dd FACES-3558 Upgrade to com.liferay.faces.test.selenium-0.5.5.jar
     [exec] Switched to branch '4.x'
     [exec] Switched to branch '4.x'
     [exec] HEAD is now at 09991d4c FACES-3558 Upgrade to com.liferay.faces.test.selenium-0.5.5.jar
     [exec] Switched to branch '4.x'
     [exec] Switched to branch '4.x'
[stopwatch] [stop.current.job: 14.637 sec]

BUILD SUCCESSFUL
Total time: 15 seconds
+ echo 'Exit shell.'
Exit shell.
[EnvInject] - Injecting environment variables from a build step.
[EnvInject] - Injecting as environment variables the properties file path '/opt/dev/projects/github/liferay-jenkins-ee/stop.properties'
[EnvInject] - Variables injected successfully.
[description-setter] Description set: &lt;strong&gt;liferay-frontend&lt;/strong&gt; - &lt;a href="https://github.com/liferay-frontend/liferay-portal/pull/923"&gt;PR#923&lt;/a&gt; - &lt;a href="https://test-1-5.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7833/jenkins-report.html"&gt;Jenkins Report&lt;/a&gt;&lt;ul&gt;&lt;li&gt;&lt;strong&gt;LocalFile.PortalSmokeUpgrade#ViewPortalSmokeArchive621021&lt;/strong&gt;&lt;ul&gt;&lt;li&gt;&lt;a href="https://testray.liferay.com/reports/production/logs/test-1-5/1616415869504/test-portal-acceptance-pullrequest%28master%29/7833/functional-upgrade-tomcat90-mysql57-jdk8/0/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive621021/index.html.gz"&gt;Poshi Report&lt;/a&gt;&lt;/li&gt;&lt;li&gt;&lt;a href="https://testray.liferay.com/reports/production/logs/test-1-5/1616415869504/test-portal-acceptance-pullrequest%28master%29/7833/functional-upgrade-tomcat90-mysql57-jdk8/0/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive621021/summary.html.gz"&gt;Poshi Summary&lt;/a&gt;&lt;/li&gt;&lt;/ul&gt;&lt;/li&gt;&lt;/ul&gt;&lt;br /&gt;
Finished: FAILURE</code></pre></div></li></ol></div></li><li><div><h5><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/332066/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>3 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/332066//consoleText">AXIS_VARIABLE=1,label_exp=!master #332066</a><pre><code>[beanshell] Retrying https://testray.liferay.com/reports/production/logs/test-1-5/1616415869504/test-portal-acceptance-pullrequest%28master%29/7833/modules-integration-mysql57-jdk8/0/1/ in 5 seconds
[beanshell] Retrying https://testray.liferay.com/reports/production/logs/test-1-5/1616415869504/test-portal-acceptance-pullrequest%28master%29/7833/modules-integration-mysql57-jdk8/0/1/ in 5 seconds
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/json.data
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-jenkins-ee
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210322065349152
     [exec] Result: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210322065349233
     [exec] Killing Chrome.
     [exec] Killing Firefox.
     [exec] chrome: no process found
     [exec] firefox: no process found
     [exec] chrome: no process found
     [exec] firefox: no process found
     [exec] Result: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210322065349392
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210322065349555
     [exec] HEAD is now at 00d824101d48 LPS-129442 sf
     [exec] Already on 'master'
     [exec] Already on 'master'
     [exec] HEAD is now at c79df14f BLADE-590 Remove unhelpful projects
     [exec] Switched to branch '2.x'
     [exec] Switched to branch '2.x'
     [exec] HEAD is now at 5570977dd FACES-3558 Upgrade to com.liferay.faces.test.selenium-0.5.5.jar
     [exec] Switched to branch '4.x'
     [exec] Switched to branch '4.x'
     [exec] HEAD is now at 09991d4c FACES-3558 Upgrade to com.liferay.faces.test.selenium-0.5.5.jar
     [exec] Switched to branch '4.x'
     [exec] Switched to branch '4.x'
[stopwatch] [stop.current.job: 19.511 sec]

BUILD SUCCESSFUL
Total time: 20 seconds
+ echo 'Exit shell.'
Exit shell.
[EnvInject] - Injecting environment variables from a build step.
[EnvInject] - Injecting as environment variables the properties file path '/opt/dev/projects/github/liferay-jenkins-ee/stop.properties'
[EnvInject] - Variables injected successfully.
[description-setter] Description set: &lt;strong&gt;liferay-frontend&lt;/strong&gt; - &lt;a href="https://github.com/liferay-frontend/liferay-portal/pull/923"&gt;PR#923&lt;/a&gt; - &lt;a href="https://test-1-5.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7833/jenkins-report.html"&gt;Jenkins Report&lt;/a&gt;
Finished: FAILURE</code></pre></div></li></ol></div></li><li>...</li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-5/jobs/test-portal-acceptance-pullrequest(master)/builds/7833/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-5/jobs/test-portal-acceptance-pullrequest(master)/builds/7833/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-5/jobs/test-portal-acceptance-pullrequest(master)/builds/7833/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

kresimir-coko commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest(master)/8060/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-16.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8060/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 9 out of 9 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 22 out of 23 jobs passed in 1 hour 30 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54">c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54">c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//1666">Acceptance Upstream DXP (master) #1666</a></p><details><summary><strong>ci:test:stable - 9 out of 9 jobs PASSED</strong></summary><details><summary><strong>9 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429444/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469533/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469536/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/5127/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/417980/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/408630/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469534/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429445/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/417981/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 21 out of 23 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest(master)/8060/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/417982/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>21 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469535/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429448/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429444/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/438000/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429450/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469533/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469536/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429451/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/5126/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429447/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/5127/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/5125/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429449/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/417980/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429446/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/408630/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469534/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/5128/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429445/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429443/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/417981/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-16.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8060/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54:</strong></summary><ol><li><div><h5><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/417982/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>1316 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/417982//consoleText">AXIS_VARIABLE=3,label_exp=!master #417982</a><ol><li><div><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/417982//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
JSON storage adapter is deprecated, using default storage adapter
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.scanXMLLogFile(PortalLogAssertorTest.java:171)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:98)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:88)
	at java.nio.file.Files.walkFileTree(Files.java:2670)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog(PortalLogAssertorTest.java:86)
</code></pre></div></li></ol></div></li></ol></div></li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-16/jobs/test-portal-acceptance-pullrequest(master)/builds/8060/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-16/jobs/test-portal-acceptance-pullrequest(master)/builds/8060/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-16/jobs/test-portal-acceptance-pullrequest(master)/builds/8060/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

jbalsas commented on modules/apps/document-library/document-library-web/.npmbundlerrc:
```diff
@@ -18,12 +18,6 @@
 				"clay-portal": ">=2.9.0",
 				"clay-radio": ">=2.9.0"
 			},
```
I think in this case all of `frontend-taglib-clay` could be removed as well since there are no references to it neither in `package.json` nor in the code in the module.

I know it might feel like some times I ask you to do the extra work and others I ask you not to... hope there's a difference between both cases. In here, this seems like useless configuration that has very minimal chance of affecting the module's behaviour... 🤞

kresimir-coko replied:
Yep, I get it @jbalsas. 👍 

Should I do this more often? By `this` i mean when touching files, to check if there are some redundancies, like in this case `frontend-taglib-clay` that could be removed. Only in very simple cases like this one, I assume.

wincent replied:
> Should I do this more often?

Attention to detail is a good thing, so it's good to check and double-check things as you go. 👍

Having said that, there are lots of "errors" that machines are very good at catching and should be automated. This might be one of those things.

jbalsas replied:
> > Should I do this more often?
>
> Attention to detail is a good thing, so it's good to check and double-check things as you go. 👍

There's also the whole [Systems Thinking for Developers](https://vimeo.com/showcase/8258302/video/520133397). Sometimes you don't have to zoom out so much to make these connections. Just looking around a little bit, understand the system you're interacting with and get a sense of things out of place.

kresimir-coko commented:
I've rebased with `master` and removed `frontend-taglib-clay` from `.npmbundlerrc` just like @jbalsas mentioned in https://github.com/liferay-frontend/liferay-portal/pull/923#discussion_r599390092

kresimir-coko commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/7940/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-14.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7940/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 9 out of 9 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 22 out of 23 jobs passed in 1 hour 48 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54">c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54">c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//1666">Acceptance Upstream DXP (master) #1666</a></p><details><summary><strong>ci:test:stable - 9 out of 9 jobs PASSED</strong></summary><details><summary><strong>9 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/325577/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/325581/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457758/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457340/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469569/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457366/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469571/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469568/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/325580/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 21 out of 23 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/7940/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221607/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>21 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/325582/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457756/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/325577/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469566/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429485/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/325581/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457758/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/325583/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457757/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469570/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457340/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/325579/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/418012/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469569/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/325578/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457366/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469571/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/418013/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469568/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/469567/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/325580/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-14.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7940/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54:</strong></summary><ol><li><div><h5><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221607/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>1316 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/221607//consoleText">AXIS_VARIABLE=3,label_exp=!master #221607</a><ol><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/221607//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
JSON storage adapter is deprecated, using default storage adapter
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.scanXMLLogFile(PortalLogAssertorTest.java:171)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:98)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:88)
	at java.nio.file.Files.walkFileTree(Files.java:2670)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog(PortalLogAssertorTest.java:86)
</code></pre></div></li></ol></div></li></ol></div></li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-14/jobs/test-portal-acceptance-pullrequest(master)/builds/7940/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-14/jobs/test-portal-acceptance-pullrequest(master)/builds/7940/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-14/jobs/test-portal-acceptance-pullrequest(master)/builds/7940/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

jbalsas commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously passed test suites:
`ci:test:relevant`

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-17.liferay.com/job/test-portal-source-format/4867/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-source-format/builds/4867/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54">c9d5d1ed47e4ae6d6b1303d3b90e485a468b6f54</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/kresimir-coko/liferay-portal/tree/LPS-125185">LPS-125185</a><br/>Branch GIT ID: <a href="https://github.com/kresimir-coko/liferay-portal/commit/6cbbe88faede7593ecb50f6cf6bedf89fac1c9fc">6cbbe88faede7593ecb50f6cf6bedf89fac1c9fc</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-17.liferay.com/job/test-portal-source-format/4867/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-17.liferay.com/userContent/jobs/test-portal-source-format/builds/4867/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pullrequest to `brianchandotcom`.

liferay-continuous-integration commented:
Pull request has been successfully forwarded to https://github.com/brianchandotcom/liferay-portal/pull/100175
